games/static/games/proto.js

```diff
@@ -325,10 +326,15 @@ class Board {
             this.updateIndexOf(rocket)
             rocketLocation = this.getIndexOf(rocket)
             nextRocketLocation = this.update(nextRocketLocation)
+            // The next if statement checks if the rocket is at the top of the board
```

Comment:
i feel this comment is nice to have but redundant 
rull of comments is: you explain the `why` and not the `how` you did it
if i read your code i can understand it right away 
(this means your code is very readable - which is a good thing)

**always prefer readable code on comments**

so - i would say, no comment is needed here